packages/x-date-pickers-pro/src/MultiInputDateTimeRangeField/MultiInputDateTimeRangeField.types.ts

```diff
@@ -28,7 +28,7 @@ export type UseMultiInputDateTimeRangeFieldComponentProps<TDate, TChildProps ext
   UseMultiInputDateTimeRangeFieldProps<TDate>;
 
 export interface MultiInputDateTimeRangeFieldProps<TDate>
-  extends UseMultiInputDateTimeRangeFieldComponentProps<TDate, TextFieldProps> {
+  extends UseMultiInputDateTimeRangeFieldComponentProps<TDate, {}> {
```

Comment:
@alexfauquette I'm not sure why this component had `TextFieldProps` in here, but it was the one, which inspired me to initially go with adding it to other multi input range components and confuse me at the end.
Did end up removing this from all components and this is the only one, which had it in the first place.

Reply:
I probably forget that props were passed to the Stack and not the TextField

`{}` could be replaced by `StackProps`

It let me realise that the default Root components does not allow to override the `spacing`, `direction`, and `alignItems`


https://github.com/mui/mui-x/blob/21afe09ac1cc2388fce45762597bda0d2110a32c/packages/x-date-pickers-pro/src/MultiInputDateTimeRangeField/MultiInputDateTimeRangeField.tsx#L11-L20

Reply:
> It let me realise that the default Root components does not allow to override the spacing, direction, and alignItems

The spread should probably be after indeed

Reply:
Will create a separate PR to address these issues. 😉